dapps-profiling/src/dapps/liquidity-pool/liquidity-pool-contract.ts

```diff
@@ -0,0 +1,128 @@
+import { ContractEngine } from "stellar-plus/lib/stellar-plus";
+import { ContractEngineConstructorArgs } from "stellar-plus/lib/stellar-plus/core/contract-engine/types";
+import { TransactionInvocation, Network } from "../../utils/simulation/types";
+import { hexStringToBytes32 } from "../../utils/converters";
+import { Address } from "@stellar/stellar-base";
+
+export enum liquidityPoolTransactions {
+    initialize = "initialize",
+    deposit = "deposit",
+    get_rsrvs = "get_rsrvs",
+    swap = "swap",
+    withdraw = "withdraw"
+}
+
+type initializeArgs = {
+    tokenWasmHash: string,
+    tokenA: string,
+    tokenB: string,
+    txInvocation: TransactionInvocation,
+}
+
+export type depositArgs = {
+    to: string,
+    desiredA: number,
+    minA: number,
+    desiredB: number,
+    minB: number,
```

Comment:
These values must be of type bigint, since they are i128 (the other methods will also need this fix)